nengo_viz/static/viz_hotkeys.js

```diff
@@ -0,0 +1,23 @@
+VIZ.Hotkeys = function () { 
+
+    document.addEventListener('keydown', function(ev) {
```

Comment:
A bit of googling tells me that we should be listening to the `'keypress'` event rather than `'keydown'`. Apparently that plays much nicer with international keyboards (see, e.g., [this SO question](http://stackoverflow.com/questions/24995316/how-to-make-difference-between-question-mark-and-for-azerty-keyboard)).